app/lib/linters/crunch_api/api_creator.rb

```diff
@@ -0,0 +1,34 @@
+module Linters
+  module CrunchApi
+
+    # Post the file to 42Crunch according to their API v1 "Create an API (from file)" spec
+    class ApiCreator
```

Comment:
Yeah something like that is a bit more descriptive and makes the return value a bit more obvious.

Reply:
Well, it does create an API per "crunch speak" and return the ID for that API... which is really the key to the object that is created rather than the object we create. So it's fine as it is in my opinion to call it api_creator. But I am also not fussed if it is changed.